astro/src/content/blog/announcing-fusionauth-1-61.mdx

```diff
@@ -0,0 +1,63 @@
+---
+publish_date: 2025-11-13
+title: Announcing FusionAuth 1.61 - The Flexible Fox
+description: "New in FusionAuth 1.61: Multiple Google IdPs per instance, SAML ForceAuthN support, and fixed MFA trust persistence. Adapt authentication to your exact needs."
+authors: Brad McCarty
+image: /img/blogs/release-1-61/fusionauth-1-61.png
+categories: Product
+tags: SAML, MFA, ForceAuthN, SSO, IdP
+excerpt_separator: "{/* more */}"
+---
+We're excited to release FusionAuth 1.61, delivering flexibility where you need it most: managing multiple identity providers, maintaining user trust through password changes, and controlling authentication timing with precision.
+
+We call this release Flexible Fox because it gives you the agility to adapt authentication to your exact needs. Foxes are known for their intelligence and ability to thrive in diverse environments. So our Flexible Fox knows exactly what you need when managing authentication across multiple tenants, testing new configurations, or balancing security with user experience.
+
+![The Flexible Fox](/img/blogs/release-1-61/fox.png)
+
+## Multiple Identity Providers, One Instance
+
+You can now configure multiple Google identity providers in a single FusionAuth instance. Each provider runs its own reconcile lambda, giving you the flexibility to handle user data differently across tenants and applications.
+
+This matters when you're running multiple environments. Testing new reconciliation logic no longer means spinning up separate instances or creating custom OIDC providers for platforms that FusionAuth natively supports.
+
+### What this means for your applications
+
+**Test without risk**
+Your staging environment can test new user attribute mapping while production continues with its proven configuration. No migration, no disruption, no compromise.
+
+**Deploy with confidence**
+When your new reconcile logic proves solid in testing, roll it to production without touching infrastructure or moving users. The transition is seamless because you're already running on the same FusionAuth instance.
+
+**Scale your way**
+Different tenants can handle Google authentication differently. Each gets the reconciliation logic that makes sense for its use case.
+
+## SAML ForceAuthN Support
+
+FusionAuth now supports SAML ForceAuthN in both identity provider and service provider configurations. This gives you control over when users must re-authenticate, even when they have an active session.
```

Comment:
Are we capitalizing the N? This is normally "ForceAuthn." Just to make sure we're consistent, I wanted to take a closer look at the capitalization. See the issue here: https://github.com/FusionAuth/fusionauth-issues/issues/2989